.gitattributes

```diff
@@ -63,3 +63,9 @@
 #*.PDF   diff=astextplain
 #*.rtf   diff=astextplain
 #*.RTF   diff=astextplain
+
+# Generated files
+src/Compilers/CSharp/Portable/Generated/*                  linguist-generated=true
+src/Compilers/CSharp/Portable/CSharpResources.Designer.cs  linguist-generated=true
```

Comment:
Why just this and not all .Designer.cs? Or should this line be pushed to linguist's actual implementation?

Reply:
I wasn't sure everything *.Designer.cs was generated.